tests/tests_pytorch/trainer/connectors/test_accelerator_connector.py

```diff
@@ -576,6 +576,7 @@ def mock_hpu_available(monkeypatch, value=True):
     monkeypatch.setattr(lightning_habana.plugins.precision, "_HPU_AVAILABLE", value)
 
 
+@pytest.mark.skipif(not package_available("lightning"), reason="Supported only with mono-package")
```

Comment:
The resolved conversation suggests reverting this. Why wasn't it done?

This is an important test, we wouldn't want to skip it on standalone

```suggestion
```

Reply:
removed